sljit_src/sljitConfigInternal.h

```diff
@@ -971,27 +971,41 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_sw sljit_exec_offset(void *code);
 #endif /* !SLJIT_COMPILE_ASSERT */
 
 #ifndef SLJIT_FALLTHROUGH
-
-#if __cplusplus >= 201703L
-#define SLJIT_FALLTHROUGH [[__fallthrough__]];
-#elif __STDC_VERSION__ >= 202311L \
-	|| (__STDC_VERSION__ > 201710L && defined(__has_c_attribute))
-#if __STDC_VERSION__ >= 202311L || __has_c_attribute(fallthrough) >= 201904L
-#define SLJIT_FALLTHROUGH [[__fallthrough__]];
+#if defined(__cplusplus) && __cplusplus >= 201703L && \
+	defined(__has_cpp_attribute)
+/* Standards-compatible C++ variant. */
+#if __has_cpp_attribute(fallthrough)
+#define SLJIT_FALLTHROUGH [[fallthrough]];
```

Comment:
isn't `[[fallthrough]]` warranted to be there by compliance with the C++17 standard?

Reply:
FWIW, sljit (although an older version) is in the NetBSD kernel and the currently supported 9.x series uses gcc 7.